### DISCUSSION TITLE
Automatic rename on import based on metadata

### DISCUSSION THREAD

**Discussion by KillahB33** [gotson/komga#1623] [Ideas]
Wondering if it's possible to do a rename on import based on Metadata.
The_Walking_Dead_090_(2011) is the format I use so I'm wondering if once I select the number from the series I want if it's possible to pull the year info from the meta data then do the rename.

I know this is a bit bigger, but series detection on import would be huge too.

**Comment by gotson** [comment]
you can already rename on import, though it's a manual operation. Series detection on import is also implemented based on book metadata (`ComicInfo.xml` or epub metadata).

**Reply by KillahB33** [reply]
Yeah manual rename works fine, just wish it was automated based on a format you provide. Ah ok so if it's not in the metadata than it won't work, I'll have to take a look to see if that's why. Thanks.
